code/headers/frame_action.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <base_comm.hpp>
+#include <hwlib.hpp>
+
+#include <cstring>
+
+namespace r2d2::robos {
+    template <frame_type Type>
+    class frame_action_c {
+    protected:
+        // set to true if the data from the frame is changed since the last call
+        // of reply_to_data
```

Comment:
I think this comment is supposed to be on `bool changed` at line 17 instead of on `base_comm_c &comm`

Reply:
Thank you for adressing this

fixed in https://github.com/R2D2-2019/RobOS/pull/3/commits/cfaff485e9ac1cc3207c3db54e5d30bd6be03395